src/virtual_dom/mod.rs

```diff
@@ -78,7 +78,7 @@ pub trait VDiff {
     type Component: Component<Self::Context>;
 
     /// Remove itself from parent and return the next sibling.
```

Comment:
nit: Remove -> Detach

Reply:
I haven't fixed the comments as not to waste time for them now. I want to fix it later.